### PR TITLE
Add model

### PR DESCRIPTION
**Pull request by Ravemaster8**
Added content to the models and added the route.

### CODE REVIEW

File: app/models/book.rb [hunk 1]
```diff
@@ -1,2 +1,12 @@
 class Book < ApplicationRecord
+    has_many :reservations
```
Comment (davidlusztig) [inline app/models/book.rb:2]: in the same time a book cannot have many reservations. please take a look at this again!

File: app/models/book.rb [hunk 2]
```diff
@@ -1,2 +1,12 @@
...
+    belongs_to :user
+
+    validates :book_title, presence: true
+    validates :isbn_number, presence: true
+    validates :book_price, presence: true
```
Comment (davidlusztig) [inline app/models/book.rb:7]: please test this with 0

**Review by davidlusztig — CHANGES_REQUESTED** [review]

File: app/models/book.rb (hunk 1 above)
Comment (Ravemaster8) [reply app/models/book.rb:2]: A book can have many reservations, for example in the next month, the book can be rented out to multiple people. This validation you are talking about, that a book can't be rented out multiple times at the same time, we'll have to validate later.

File: app/models/book.rb (hunk 2 above)
Comment (Ravemaster8) [reply app/models/book.rb:7]: Tested and confirmed it works with 0.

**Pull request closed without merging**